recipes-core/cvm-reverse-proxy-client/files/cvm-reverse-proxy-client-init

```diff
@@ -15,28 +15,40 @@ DESC="CVM Reverse Proxy Client"
 DAEMON_ARGS="--listen-addr=localhost:7937 --target-addr=https://hub-atls.builder.flashbots.net --client-attestation-type azure-tdx --server-attestation-type none"
 PIDFILE=/var/run/$NAME.pid
 LOGFILE=/var/log/$NAME.log
+SYSTEM_API_FIFO=/var/volatile/system-api.fifo
+
+log() {
+    if [ -p $SYSTEM_API_FIFO ]; then
+        date_log() {
+            echo -n ""
+        }
+    else
+        date_log() {
+            echo -n "$(date): "
+        }
+    fi
+    echo "$(date_log)$1" | tee -a $SYSTEM_API_FIFO
+}
```

Comment:
same comment as in the other [PR](https://github.com/flashbots/meta-evm/pull/44/files#r1862213989)